src/scripts/main.js

```diff
@@ -1 +1,94 @@
 'use strict';
+
+// const firstPromise = new Promise((resolve, reject) => {
+//   let isTrue = false;
+
+//   setTimeout(() => {
+//     document.addEventListener('click', () => {
+//       isTrue = true;
+//       resolve('First promise was resolved');
+//     });
+//   }, 3000);
+
+//   if (isTrue === false) {
+//     reject(new Error('First promise was rejected'));
+//   }
+// });
+async function wait(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+const firstPromise = new Promise(async (resolve, reject) => {
+  let isTrue = false;
+
+  document.addEventListener('click', () => {
+    isTrue = true;
+    resolve('First promise was resolved');
+  });
+
+  await wait(3000);
+
+  if (isTrue === false) {
+    reject(new Error('First promise was rejected'));
+  }
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  let isTrue = false;
+
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+    isTrue = true;
+  });
+
+  if (isTrue !== true) {
```

Comment:
The check `isTrue !== true` is unnecessary. Since `isTrue` is a boolean, you can simply write `if (!isTrue)`. This would make your code cleaner and easier to read.